physionet-django/user/migrations/0050_auto_20221216_1547.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.14 on 2022-12-16 20:47
```

Comment:
This migration can be removed

Reply:
Thank you. Removing them. i was also wondering the same and asked Harry if that would be okay